symengine/dict.h

```diff
@@ -42,6 +43,7 @@ typedef std::map<RCP<const Integer>, unsigned,
         RCPIntegerKeyLess> map_integer_uint;
 typedef std::map<unsigned, integer_class>
        map_uint_mpz;
+typedef std::map<unsigned, Expression> map_uint_Expr;
```

Comment:
I think that's fine. Open to change though.

On Thu, Mar 17, 2016 at 2:42 PM, Akash Trehan <notifications@github.com>
wrote:

> In symengine/dict.h
> <https://github.com/symengine/symengine/pull/810#discussion_r56582617>:
>
> > @@ -42,6 +43,7 @@ typedef std::map<RCP<const Integer>, unsigned,
> >          RCPIntegerKeyLess> map_integer_uint;
> >  typedef std::map<unsigned, integer_class>
> >         map_uint_mpz;
> > +typedef std::map<unsigned, Expression> map_uint_Expr;
>
> Another thing....UnivariateIntPolynomial still uses unsigned int.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/symengine/symengine/pull/810/files/bc5ee19d010874f201ccd2b5219bb583bb687699#r56582617>
>